src/goto-diff/goto_diff_parse_options.cpp

```diff
@@ -335,9 +335,13 @@ int goto_diff_parse_optionst::doit()
     return 0;
   }
 
-  if(cmdline.isset("change-impact"))
+  if(cmdline.isset("change-impact") || cmdline.isset("forward-impact") || cmdline.isset("backward-impact"))
```

Comment:
Apologies in advance for nit-picking, but to keep the code base clean: lines should, where reasonably possible, be no longer than 70 characters.